yarn.lock

```diff
@@ -4,20 +4,20 @@
 
 "@aashutoshrathi/word-wrap@^1.2.3":
   version "1.2.6"
-  resolved "https://registry.yarnpkg.com/@aashutoshrathi/word-wrap/-/word-wrap-1.2.6.tgz#bd9154aec9983f77b3a034ecaa015c2e4201f6cf"
+  resolved "https://registry.npmjs.org/@aashutoshrathi/word-wrap/-/word-wrap-1.2.6.tgz"
```

Comment:
Looks like the registry changed from `yarnpkg.com` -> `npmjs.org` with the package installation. Would be a smaller diff if the registry remained `yarnpkg.com`. I'm not sure off the top of my head how to customise the registry, but [this post](https://stackoverflow.com/questions/58071109/difference-between-yarn-registry-and-npm-registry) gives a pointer.

Reply:
I wonder if we should switch to `pnpm` at some point. I get the impression that yarn v1 is slowly falling out of favour but I'm not sure how easy it is to modify code in `node_modules` directly with `pnpm` and we tend to do that quite a bit when debugging updated dependencies.

Reply:
I've found `pnpm` to be really quite nice as it installs dependencies quickly, uses YAML for its lockfile, and plays very nicely with monorepos (its workspace implementation is really nice). But it's true that the [dependency tree](https://pnpm.io/symlinked-node-modules-structure) is head-spinning if you ever have to look under the hood.

I don't fully understand it. It mentions a store, which makes one think of a central store in one's user directory, but as far as I can tell from following the symlinks in a project of my own, it's only symlinking back to a store under the project directory rather than under your user directory, so you can't accidentally leave some debug logs in one project and accidentally end up affecting another on your system.

So the main downside is the really long file paths that you see in your IDE; it's easy to get lost when doing this kind of debugging, compared to other package managers.

But I'd say it's a decent tradeoff. `npm` and `yarn` installs feel like they take ages. `bun` isn't really in the running for my so long as they continue to use a binary lockfile.

Reply:
I reverted the registry to `yarnpkg.com`, but the diff might be the same size still, at least in terms of lines, since it knocked the commit hash off the end of every `resolved...`entry. Not sure, though.

Reply:
Per conversation above I've removed the `@stylistic/eslint-plugin-js` dependency and the max-len rule ESLint rule, so the yarn.lock is unchanged now.